packages/shared/lib/contexts/governance/actions/registerProposalsFromNode.ts

```diff
@@ -3,22 +3,11 @@ import { get } from 'svelte/store'
 import { selectedAccount } from '@core/account/stores'
 import { INode } from '@core/network/interfaces'
 
-import { getVotingParticipationEventIds, isProposalAlreadyAddedForSelectedAccount } from '../utils'
-
-import { registerParticipationEvent } from './registerParticipationEvent'
+import { registerParticipationEvents } from './registerParticipationEvents'
 
 export async function registerProposalsFromNode(node: INode): Promise<void> {
-    const proposalIds = await getVotingParticipationEventIds(node)
-    if (!proposalIds || proposalIds.length === 0) {
-        return
+    const options = {
+        node,
     }
-
-    const _selectedAccount = get(selectedAccount)
-    await Promise.all(
-        proposalIds.map(async (proposalId) => {
-            if (!isProposalAlreadyAddedForSelectedAccount(proposalId)) {
-                await registerParticipationEvent(proposalId, node, _selectedAccount)
-            }
-        })
-    )
+    await registerParticipationEvents(options, get(selectedAccount))
```

Comment:
Why is this for the selected account? We will need to register events for any account.